Hazelcast.Test/Hazelcast.Client.Test/ClientRetryTest.cs

```diff
@@ -177,5 +179,58 @@ public void TestRetryTimeout()
                 client.Shutdown();
             }
         }
+
+ 
+        [Test, MaxTime(60000)]
+        public void TestRetry_Owner_changed()
+        {
+            var dict = new Dictionary<string, string>
+            {
+                {"1", "1"},
+                {"2", "2"},
+                {"3", "3"},
+                {"4", "4"},
+                {"5", "5"}
+            };
+
+            var member = _remoteController.startMember(_cluster.Id);
+            var client = CreateClient();
+            var member2 = StartMemberAndWait(client, _remoteController, _cluster, 2);
+            var map = client.GetMap<string, string>(TestSupport.RandomString());
+            map.PutAll(dict);
+
+            var terminate = new ManualResetEvent(false);
+            var task = Task.Factory.StartNew(() =>
+            {
+                while (!terminate.WaitOne(0))
+                {
+                    var st = new Stopwatch();
+                    st.Start();
+                    var result = map.GetAll(dict.Keys);
+                    var took = st.ElapsedMilliseconds;
+                    if (took > 1500)
```

Comment:
Time sensitive testing may be problem at different environments especially when too many processes running on the same machine (e.g. parallel dockers, we observed during compatibility test runs).

Reply:
This is a delay measurement that the theoretical max is 10 sec and we want it as fast as 100 ms or so. I'm assign a comparibly a large number. I don't know a better solution for that.